packages/storage/src/implementation/error.ts

```diff
@@ -54,7 +54,11 @@ export class FirebaseStorageError implements Error {
   }
 
   get message(): string {
-    return this.message_;
+    if (this.serverResponse_) {
+      return this.message_ + '\n' + this.serverResponse_;
```

Comment:
Added this so that the server response shows up in the error message for "unknown" errors.